src/contexts/json-rpc/json-rpc-provider.tsx

```diff
@@ -15,6 +15,8 @@ const createClient = (logger: ReturnType<typeof useLogger>) => {
       backgroundPort.postMessage(msg)
     }
   })
+  // @ts-ignore
```

Comment:
If we do this: https://github.com/vegaprotocol/vegawallet-browser/pull/208/files#r1196364639

we can remove this

```suggestion
```